lib/private_key.ml

```diff
@@ -18,24 +18,14 @@ module Asn = struct
         (required ~label:"exponent"    integer)
```

Comment:
further above (lines 1+2), the `let src = ...` and `module Log = ...` can be removed as well.